packages/fxa-shared/metrics/validate.ts

```diff
@@ -178,14 +178,12 @@ export class MetricValidator {
   }
 
   /**
-   * Check for bad potentially bad L1 navigation timing values.
+   * Check for bad potentially bad navigation timing values.
    */
-  public hasInvalidL1TimingData(
-    data: Pick<CheckedMetrics, 'navigationTiming'>
-  ) {
+  public hasInvalidTimingData(data: Pick<CheckedMetrics, 'navigationTiming'>) {
     // Check for magic numbers that indicate bad l1 navigation timing data.
```

Comment:
Q: Does the comment need to change here to say "L1 or L2" or more generic "bad navigation timing" (like above on L181)?